engine/src/main/java/org/terasology/config/flexible/constraints/NumberRangeConstraint.java

```diff
@@ -13,32 +13,34 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.terasology.config.flexible.validators;
+package org.terasology.config.flexible.constraints;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.terasology.config.flexible.Setting;
 
 /**
- * Validates a {@link Number} within the specified range.
+ * Constrains a {@link Number} within the specified range in a {@link Setting}.
  *
- * @param <T> The type of the {@link Number} to validate.
+ * @param <T> The type of {@link Number} to constrain.
  */
-public class RangedNumberValidator<T extends Number & Comparable<? super T>> implements SettingValueValidator<T> {
-    private static final Logger LOGGER = LoggerFactory.getLogger(RangedNumberValidator.class);
+public class NumberRangeConstraint<T extends Number & Comparable<? super T>> implements SettingConstraint<T> {
+    private static final Logger LOGGER = LoggerFactory.getLogger(NumberRangeConstraint.class);
```

Comment:
For consistency with the codebase, let's use logger? Not an issue one way or the other though.

Reply:
Again, the issue of legacy vs conventions :) Java convention calls for class constants (`static final` fields) to be written in `UPPER_SNAKE_CASE`, but Terasology has largely used `logger` instead of `LOGGER`. It must be pointed out though that some existing classes in the code base also use `LOGGER` instead of `logger`, and since this is a private constant (and you don't mind leaving it as is), I'll opt to not make an extra change.